server/src/main/java/com/saralein/server/response/Response.java

```diff
@@ -1,14 +1,17 @@
 package com.saralein.server.response;
 
 import com.saralein.server.exchange.Header;
+import com.saralein.server.exchange.Message;
 
 public class Response {
     private final Header header;
     private final byte[] body;
+    private final Message message;
 
-    public Response(Header header, byte[] body) {
+    public Response(Header header, byte[] body, Message message) {
```

Comment:
I don't think I understand this. What is the `Message` here? How is that related to `header` and `body`?

Reply:
After our conversation, going to close this PR.